test/functional/specs/Data Collector/C8118.js

```diff
@@ -33,6 +33,7 @@ test("Test C8118: Load page with link. Click link. Verify event.", async () => {
 
   await t.click(Selector("#alloy-link-test"));
   await t.expect(getLocation()).contains("blank.html");
+  await t.expect(networkLogger.edgeCollectEndpointLogs.requests.length).eql(1);
```

Comment:
This is unrelated to the test being fixed in the PR. In this case, the test is failing when run against prod as expected because Alloy prod doesn't handle sendBeacon properly. However, the failure message was `TypeError: Cannot read property 'request' of undefined` because we try to access `networkLogger.edgeCollectEndpointLogs.requests` on the line below. I decided to add this check for length just to make the failure message easier to understand.